src/chariot/database/MySQLDatabaseWriter.py

```diff
@@ -0,0 +1,60 @@
+from typing import List, Tuple
+import mysql.connector as connector
+
+from chariot.database.DatabaseWriter import DatabaseWriter
+
+
+class MySQLDatabaseWriter(DatabaseWriter):
+    def __init__(self, connectionString: str):
```

Comment:
make connectionString more configurable, also think about how choosing a table would work in the configuration.

Reply:
Turn to the way it's done for devices on how to properly implement configurations.